README.md

```diff
@@ -1 +1,62 @@
 # Manifest Schema
+
+This project contains publicly available json schemas that define slacks `manifest.json` file. JSON schemas can be used for a number of tasks but the purpose of this specific schema is to provide developers validation and prediction capabilities while editing `manifest.json` files in IDEs.
+
+**Guide Outline**:
+
+- [Manifest Schema](#manifest-schema)
+  - [Usage](#usage)
```

Comment:
I would also add a "Running Tests" section that explains how to run the tests in the repo (so that if I am contributing or playing around with it, I have instructions to make sure the project is working as expected before/after I make any changes).

Reply:
I've placed the instructions of how to run the tests in the `.github/maintainers_guide.md` since they should not be used by users of the `schemas` they are mainly for the CI pipeline and the maintainers.

Let me know what you think